exercise1.R

```diff
@@ -1,5 +1,27 @@
+library(ggplot2)
+
 ba = read.table("https://raw.githubusercontent.com/OxfordIHTM/teaching_datasets/main/ba.dat",header=TRUE)
 
+average = rowMeans(ba)
+ba$average = average
+#Mean differences 
+difference <- ba$Wright - ba$Mini
+ba$difference = difference
+mean_difference = mean(difference)
+
+#Confidence interval
+lower <- mean_difference - 1.96*sd(ba$difference)
+upper <- mean_difference + 1.96*sd(ba$difference)
```

Comment:
Good job here! Well done! This si a good example of a scripts-based approach to calculating all the elements that you need for creating a Bland and Altman plot.

I will add it here as a solution but this doesn't fully answer my task/issue because I was asking for a function. And the examples from Mariano and Clifford below is what I was really looking for.